src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -463,20 +464,6 @@
                     </div>
                 </div>
             </div>
-            <div id="print-alert-container" class="print-alert">
-                <h3>
-                    Your browser does not automatically detect page layout. Check your print settings to make sure page layout is set correctly.
-                </h3>
-                <div class="row">
-                    <div class="left-col">
-                        <div class="export-output-area">
-                            <a href="#" id="print-alert-okay" class="button button-primary radius">Okay</a>
-                        </div>
-                    </div>
-                    <div class="right-col">
-                    </div>
-                </div>
-            </div>
```

Comment:
Consider squashing these commits since the second commit reverts things that were added in the first commit.